package.json

```diff
@@ -72,8 +71,8 @@
     "fs-extra": "^8.1.0",
     "glob-parent": "^5.1.0",
     "globby": "^9.2.0",
-    "jest": "^24.9.0",
-    "jest-circus": "^24.9.0",
+    "jest": "^26.2.2",
```

Comment:
ahhh, I missed this in my look-through; my apologies. Looks like this is good to go, then, I'm thinking?